specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2024-05-15/managedCassandra.json

```diff
@@ -0,0 +1,1426 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "CassandraDataCenters_CreateUpdate",
+        "description": "Create or update a managed Cassandra data center. When updating, overwrite all properties. To update only some properties, use PATCH.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraDataCenterCreate": {
+            "$ref": "./examples/CosmosDBManagedCassandraDataCenterCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            },
+            "description": "Parameters specifying the managed Cassandra data center."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Accepted. The data center will be created asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            }
+          },
+          "200": {
+            "description": "Accepted. The data center will be updated asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "CassandraDataCenters_Update",
+        "description": "Update some of the properties of a managed Cassandra data center.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraDataCenterUpdate": {
+            "$ref": "./examples/CosmosDBManagedCassandraDataCenterPatch.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v2 [](http://example.com/codeflow?start=69&length=2)

Can you update to V5s through out ?

Reply:
Will update these as a separate cleanup of the whole folder in the next API version. This piece of code is not updated in this version.